### DISCUSSION TITLE
Actividad 4 - GRUPO 1

### DISCUSSION THREAD

**Discussion by Fedebravo12** [frlp-utn-ingsoft/catedra#151] [Show and tell]
https://github.com/Fedebravo12/vetsoft.git

**Comment by ivanduva** [comment]
Buenas! Les puedo pedir el link a donde lo deployaron?

**Reply by germansalinas1994** [reply]
https://vetsoft-app-v1-1-1.onrender.com/

**Comment by ivanduva** [comment]
Perfecto.
La actividad está muy bien, queda aprobada. Redondearon un muy buen trabajo en términos generales.
Como puntos altos podemos mencionar la calidad de los tests en general, el buen trabajo en los PRs y la mejora del código.
Les dejamos unos comentarios sobre cuestiones que vimos que si bien son menores, no queriamos dejar de mencionar para que les sirva en el futuro:

- Ojo cuando commitean muchos archivos a la vez. Vimos que en un PR subieron un archivo de configuración de VS code. Ese tipo de cosas se pueden agregar al `.gitignore`. Pero igualmente tengan cuidado al hacer `git add .` (esto es una mala práctica) o hacer `git add `sobre carpetas.
- Vimos algunos tests de unidad e integración que en lugar de testear que salte el error, testean si se creo o no la instancia en la lista de objetos. Eso es muy poco específico, podría haber fallado la creación por cualquier razón. En su lugar deberían testear que efectivamente el fallo sea porque el campo es incorrecto. Por ejemplo en el caso de los tests del campo mail o name.

Buen trabajo!
Saludos
